website/docs/reference/directives.md

```diff
@@ -71,6 +71,17 @@ Add Java options
 #### Examples
 `using javaOpt -Xmx2g -Dsomething=a`
 
+### Main class
+
+Specify default main class
+
+`using main-class `_main class_
+
+`using mainClass `_main class_
+
+#### Examples
+`using main-class helloWorld`
+
```

Comment:
Why does this get changed in this PR? It seems none of the other changes are related to that.

Reply:
This was probably because I didn't rebase to master